### ISSUE TITLE
Suggestion : larger team selector

### ISSUE THREAD

**Issue by DaveInDev** [MEEPofFaith/testing-utilities-java#30]
Hi  MEEPofFaith, 
I use your tool a lot for map creation. A great help, thanks.

To toggle quickly between sharded/yellow and crux/red team, you just have to short-click once on the team button : perfect.

But when you make a map with more than 1 enemy (outside PVP, you can use 4 enemy teams + derelict), it's not easy to quickly change your team (red, green, blue, purple, grey) for testing and building purposes : you have to press long to make the  big list appear and then make your selection.

If you find it useful and doable, it would be nice to have a more complete quick selector like the on ein the editor, like on this montage :
![Clipboard-3](https://user-images.githubusercontent.com/20345939/219851640-6b53d4ea-8bde-4198-8142-fae7470df085.jpg)

The "long press" list could still appear on a long click.

Have a nice day.

**Comment by MEEPofFaith** [comment]
Why didn't I ever think of shoving editing mode's team changer into there

**Comment by DaveInDev** [comment]
> Why didn't I ever think of shoving editing mode's team changer into there

By curiosity, what do you use your tool for ? Map making ? Or sandbox tests ?

**Comment by Slotterleet** [comment]
As a TU user, I can say for sure that this mod is extremely useful at debugging own mods.
tl;dr: sandbox tests

**Comment by MEEPofFaith** [comment]
...It's a bit small.
![2023_02_20_22_02_36_Mindustry](https://user-images.githubusercontent.com/54301439/220263084-e565a461-eee1-407f-840e-6e07d54bf160.png)

**Comment by DaveInDev** [comment]
> ...It's a bit small. 

yes, but I love it. And also the health buttons that changed from top to bottom. Thanks.
